app/governance/components/ProposalTable/ProposalTable.tsx

```diff
@@ -43,18 +47,26 @@ const ProposalTable = ({ proposals }: TableProps) => {
     setCurrentPage(1);
     return proposals.filter((proposal) => {
       switch (currentFilter) {
-        case ProposalFilter.ACTIVE:
-          return proposal.status === "PROPOSAL_STATUS_VOTING_PERIOD";
         case ProposalFilter.PASSED:
           return proposal.status === "PROPOSAL_STATUS_PASSED";
         case ProposalFilter.REJECTED:
           return proposal.status === "PROPOSAL_STATUS_REJECTED";
         default:
-          return true;
+          return (
+            proposal.status === "PROPOSAL_STATUS_REJECTED" ||
+            proposal.status === "PROPOSAL_STATUS_PASSED"
+          );
       }
     });
   }, [currentFilter, proposals]);
 
+  const activeProposals = useMemo(() => {
```

Comment:
Added active Proposals separately as we're rendering them in a different table now